assets/javascripts/modules/checkout/promoCode.js

```diff
@@ -82,8 +81,8 @@ define([
     }
 
     function validate() {
-        var country = countryChoice.getCurrentCountryOption().value.trim(),
-            promoCode  = $inputBox.val().trim();
+        var country = formElements.BILLING.$COUNTRY_SELECT.val().trim(),
```

Comment:
Just double check whether this should be delivery or billing address - i.e. do whatever Membership does.